From 956cc06a266b600279d33d8a8363a574803e5a09 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Mon, 14 Jun 2021 16:11:25 +0200
Subject: [PATCH] Export aws_mqtt_client_connection_reconnect as part of the
 API

---
 include/aws/mqtt/client.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/aws/mqtt/client.h b/include/aws/mqtt/client.h
index 7fec354..6f0240a 100644
--- a/include/aws/mqtt/client.h
+++ b/include/aws/mqtt/client.h
@@ -416,6 +416,7 @@ int aws_mqtt_client_connection_connect(
  * \returns AWS_OP_SUCCESS if the connection has been successfully initiated,
  *              otherwise AWS_OP_ERR and aws_last_error() will be set.
  */
+AWS_MQTT_API
 int aws_mqtt_client_connection_reconnect(
     struct aws_mqtt_client_connection *connection,
     aws_mqtt_client_on_connection_complete_fn *on_connection_complete,
-- 
2.30.1 (Apple Git-130)

